ivy/functional/ivy/nest.py

```diff
@@ -1015,6 +1015,49 @@ def nested_map(
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
 
+    Examples
+    --------
+    With :code:`ivy.Array` inputs:
+
+    >>> def special_square(nested : float) -> float : return np.square(x)
+    >>> nested = ivy.Array([[1,2,3],[4,5]])
```

Comment:
Two comments here: (a) Ivy arrays are called using `ivy.array` instead of `ivy.Array` (lower "a"), and (b) the array has an non homogeneous shape so this would throw an error, you can use `ivy.array([[1,2,3],[4,5,6]])` instead if you want

Reply:
changed to ivy.array